app/scripts/views/settings/clients.js

```diff
@@ -105,6 +107,18 @@ define(function (require, exports, module) {
       });
     },
 
+    _getPanelTitle: function () {
+      var title = t('Devices');
+
+      if (this._able.choose('appsListVisible', {
```

Comment:
The code here should do it: https://github.com/mozilla/fxa-content-experiments/pull/40/files#diff-ef4b3a9aa6190a2bd91842a4370aabddR9